libsolidity/lsp/FileRepository.cpp

```diff
@@ -51,6 +67,8 @@ string FileRepository::sourceUnitNameToUri(string const& _sourceUnitName) const
 		return "file:///" + _sourceUnitName;
 	else if (_sourceUnitName.find("/") == 0)
 		return "file://" + _sourceUnitName;
+	else if (auto const resolvedPath = tryResolvePath(_sourceUnitName); resolvedPath.message().empty())
+		return "file://" + resolvedPath.get().generic_string();
 	else
 		return "file://" + m_basePath.generic_string() + "/" + _sourceUnitName;
```

Comment:
Does `generic_string()` always strip the trailing `/`? What happens when base path is `/`? Won't the resulting path be interpreted as UNC path giving weird results?

Reply:
I just tried that locally in my browser and having too many leading slashes in your file-URI works. FF simply ignores, Chrome does rewrite the URL.